.github/CODEOWNERS

```diff
@@ -19,80 +18,105 @@
-/xml/System.ComponentModel.DataAnnotations**/** @lajones @ajcvickers
-/xml/System.ComponentModel.Design**/** @RussKie @merriemcgaw
-/xml/System.Composition**/** @maryamariyan @ViktorHofer
-/xml/System.Configuration**/** @maryamariyan @safern
-/xml/System.Data**/** @ajcvickers @cheenamalhotra @david-engel
-/xml/System.Data.SqlClient**/** @cheenamalhotra @david-engel @karinazhou @JRahnama
-/xml/System.Diagnostics**/** @tommcdon @sywhang @krwq
-/xml/System.Diagnostics/DiagnosticSource.xml @Anipik
-/xml/System.Diagnostics/EventLog*.xml @Anipik
-/xml/System.Diagnostics/PerformanceCounter*.xml @Anipik
-/xml/System.Diagnostics/Process*.xml @adsitnik @eiriktsarpalis
-/xml/System.Diagnostics/TraceSource.xml @Anipik
-/xml/System.Diagnostics.Tracing**/** @noahfalk @tommcdon @tarekgh @Anipik
-/xml/System.DirectoryServices**/** @tquerec @josephisenhour
-/xml/System.Drawing**/** @safern @tannergooding
-/xml/System.Dynamic**/** @cston @333fred
-/xml/System.Globalization**/** @safern @krwq @tarekgh
-/xml/System.IO**/** @carlossanlop @jozkee
-/xml/System.IO.Compression**/** @carlossanlop @ericstj
-/xml/System.IO.Pipelines**/** @davidfowl @halter73 @jkotalik @anurse
-/xml/System.Linq**/** @eiriktsarpalis @adamsitnik
-/xml/System.Linq.Expressions**/** @cston @333fred
-/xml/System.Linq/Parallel*.xml @tarekgh @kouvel
-/xml/System.Management**/** @Anipik
-/xml/System.Net**/** @dotnet/ncl
-/xml/System.Numerics**/** @tannergooding @pgovind
-/xml/System.Reflection**/** @steveharter @GrabYourPitchforks
-/xml/System.Reflection.Emit**/** @steveharter @GrabYourPitchforks
-/xml/System.Reflection.Metadata**/** @tmat
-/xml/System.Resources**/** 	@buyaa-n @tarekgh @krwq
-/xml/System.Runtime**/** @bartonjs @joperezr
-/xml/System.Runtime.Caching**/** @StephenMolloy @HongGit
-/xml/System.Runtime.CompilerServices**/** @Anipik @steveharter
-/xml/System.Runtime.InteropServices**/** @AaronRobinsonMSFT @jkoritzinsky
-/xml/System.Runtime.Intrinsics**/** @tannergooding @CarolEidt @echesakovMSFT
-/xml/System.Security**/** @bartonjs @GrabYourPitchforks @krwq
-/xml/System.ServiceModel**/** @HongGit @mcconnew StephenMolloy
-/xml/System.ServiceProcess**/** @Anipik
-/xml/System.Text**/** @tarekgh @GrabYourPitchforks
-/xml/System.Text/Encoding*.xml @layomia @krwq @tarekgh
-/xml/System.Text.Encodings.Web**/** @GrabYourPitchforks @layomia @tarekgh
-/xml/System.Text.Json**/** @layomia @steveharter @jozkee
-/xml/System.Text.RegularExpressions**/** @pgovind  @eerhardt
-/xml/System.Threading**/** @kouvel
-/xml/System.Threading.Channels**/** @tarekgh @stephentoub
-/xml/System.Threading.Tasks**/** @tarekgh @stephentoub
-/xml/System.Transactions**/** @dasetser @HongGit
-/xml/System.Web**/** @scottaddie
-/xml/System.Windows**/** @dotnet/wpf-developers
-/xml/System.Windows.Forms**/** @RussKie @merriemcgaw
-/xml/System.Xaml**/** @dotnet/wpf-developers
-/xml/System.Xml**/** @buyaa-n @krwq
+/xml/Microsoft.CSharp*/ @cston @333fred
+/xml/Microsoft.Extensions*/ @maryamariyan @michaelgsharp @safern @tarekgh
```

Comment:
this would need to be updated every time area-owners.md is changed. Any chance we could consolidate into one document? (e.g. by adding a column linking to the relevant docs folders in area-owners.md)

cc @carlossanlop @jeffhandley

Reply:
I will leave it up to you and Carlos to decide how reviewers are tagged. If we decide to rely on fabricbot, I think we should just delete this file.